test/testinput/bufr_ncep_highRes_sonde.yaml

```diff
@@ -0,0 +1,209 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+# 
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "./gdas.t12z.adpupa_nc002103.tm00.bufr_d"
+
+      exports:
+        #subsets:
+        #  - NC002101
+        group_by_variable: pressure
+        variables:
+          # MetaData
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+          reportIdentifier:
+            query: "*/RPID"
+          latitude:
+            query: "*/CLATH"
+          longitude:
+            query: "*/CLONH"
+          stationElevation:
+            query: "*/HSMSL"
+          stationIdentification:
+            query: "[*/IDLSIPTM/SMID, NC002104/SMID]"
+          cloudType:
+            query: "[*/CLINRVSD/CLTP[1], */CLINRVSD/CLTP[2], */CLINRVSD/CLTP[3]]"
+
+          # ObsValue
+          pressure:
+            query: "*/UARLVB/PRLC"
+          airTemperature:
+            query: "*/UARLVB/UATMP/TMDB"
+          dewpointTemperature:
+            query: "*/UARLVB/UATMP/TMDP"
+          windDirection:
+            query: "*/UARLVB/UAWND/WDIR"
+          windSpeed:
+            query: "*/UARLVB/UAWND/WSPD"
+
+          # ObsValue - ocean
+          waterTemperature:
+            query: "*/SST1"
+
+          # ObsValue - cloud
+          cloudAmount:
+            query: "*/CLINRVSD/CLAM"
+          heightOfBaseOfCloud:
+            query: "*/CLINRVSD/HOCB"
+          verticalSignificanceSurfaceObservations:
+            query: "[*/CLINRVSD/VSSO[1], */CLINRVSD/VSSO[2]]"
+
+          # QualityMarker
+          stationElevationQM:
+            query: "*/QCEVR"
+          pressureQM:
+            query: "*/UARLVB/QMPR"
+          airTemperatureQM:
+            query: "*/UARLVB/UATMP/QMAT"
+          moistureQM:
+            query: "*/UARLVB/UATMP/QMDD"
+          windQM:
+            query: "*/UARLVB/UAWND/QMWN"
+
+
+    ioda:
+      backend: netcdf
+      obsdataout: "../testoutput/gdas.t12z.adpupa_bufr_nc002103.tm00.nc"
+
+      dimensions:
+        - name: RadiosondeReportLevelData
+          path: "*/UARLVB"
+        - name: CloudInformation
+          path: "*/CLINRVSD"
+        - name: IdentificationOfLaunchSite
+          path: "*/IDLSIPTM"
+
+      variables:
+        # MetaData
+        - name: "MetaData/dateTime"
+          coordinates: "longitude latitude"
+          source: variables/timestamp
+          longName: "Datetime"
+          units: "seconds since 1970-01-01T00:00:00Z"
+
+        - name: "MetaData/reportIdentifier"
```

Comment:
Since this is not in the ObsSpace.yaml, should it be?

Reply:
@nicholasesposito fixed. There is no sondeReportIdentifier in ObsSpace.yaml. It is replaced with the stationIdentification.